BUILD

```diff
@@ -0,0 +1,15 @@
+load("@npm_bazel_typescript//:index.bzl", "ts_config")
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["tsconfig.json"])
+
+
+ts_config(
```

Comment:
Since this level of the Bazel package hierarchy doesn't exist internally, do we know how we're going to sync this and the other files?  Or do we not use any of these internally?

Reply:
> Or do we not use any of these internally?

This :)